WcaOnRails/app/views/competitions/index.html.erb

```diff
@@ -1,17 +1,144 @@
 <% provide(:title, 'Competitions') %>
 
 <div class="container">
-  <h1><%= yield(:title) %></h1>
-
-  <%= link_to "New competition", new_competition_path %>
-  <table class="table">
-    <% @competitions.each do |competition| %>
-      <tr>
-        <td><%= competition.id %></td>
-        <td><%= competition.name %></td>
-        <td><%= link_to "Edit", admin_edit_competition_path(competition) %></td>
-        <td><%= link_to "Results", "/results/competition.php?competitionId=#{competition.id}" %></td>
-      </tr>
+  <h2><%= yield(:title) %></h2>
+
+  <%= form_tag competitions_path, method: :get, class: "competition-select" do %>
+    <% if @competitions.length < 1 %>
+      <%= render 'index_form' %>
+      <div class="alert alert-danger">No competitions found!</div>
+    <% else %>
+      <% if params[:commit] == "List" %>
+        <table class="table competitions-table table-striped table-hover table-condensed floatThead">
+          <thead>
+            <tr>
+              <th colspan="7">
+                <%= render 'index_form' %>
+              </th>
+            </tr>
+            <tr>
+              <th>Year</th>
+              <th>Date</th>
+              <th>Name</th>
+              <th></th>
+              <th>Country, City</th>
+              <th>Venue</th>
+              <th class="big-column"></th>
+            </tr>
+          </thead>
+          <tbody>
+            <% @competitions.each_with_index do |competition, index| %>
+              <% if index == @closest_index %>
+                <tr id="today-line">
+                  <td colspan="6" class="text-center">Today</td>
+                  <td></td>
+                </tr>
+              <% end %>
+              <tr class="<%= competition.is_over? ? "past" : "future" %>">
```

Comment:
I thought we discussed this and were going to go with `past` and `not-past` or something? A competition that is not yet over could be ongoing.